hadoop-project/pom.xml

```diff
@@ -1275,6 +1278,12 @@
         <groupId>com.fasterxml.jackson.module</groupId>
         <artifactId>jackson-module-jaxb-annotations</artifactId>
         <version>${jackson2.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+          </exclusion>
```

Comment:
why this exclusion is required?

Reply:
Hadoop loads this anyway. The issue is that the dependency version convergence checks fail the build if transitive dependency versions to not match the explicit dependency version.

Reply:
but I didn't see the excluded dep in the list
https://mvnrepository.com/artifact/com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.14.3

do you mean `jakarta.xml.bind:jakarta.xml.bind-api`?

Reply:
Have a look at https://github.com/search?q=repo%3Aapache%2Fhadoop%20javax.xml.bind&type=code

javax.xml.bind is still used in Hadoop.

I did not add this exclusion on a whim. I added it because the Haddop build failed with a dreaded dependency convergence exception.

The tests pass with this change.

Reply:
I'll remove the exclusion and see if the dependency convergence exception still occurs. You are right that jackson-module-jaxb-annotations:2.14.3 depends on the jakarta variant of the jar.

Reply:
I added the exclusion when testing with jackson 2.18.2. 

https://mvnrepository.com/artifact/com.fasterxml.jackson.module/jackson-module-jaxb-annotations/2.18.2

that has a dependency on javax.xml.bind:jaxb-api not jakarta.xml.bind:jakarta.xml.bind-api 

jakarta.xml.bind:jakarta.xml.bind-api  v2 uses the same package name as javax.xml.bind:jaxb-api (a javax package name) - it was only in v3 that the package name changed to a jakarta package name.

I think I will add back the exclusion but fix it so that it excludes jakarta.xml.bind:jakarta.xml.bind-api.